app/src/main/kotlin/com/wire/android/ui/home/conversations/search/InternalContactSearchResultItem.kt

```diff
@@ -151,7 +151,7 @@ fun ExternalContactSearchResultItem(
                 ConnectionState.BLOCKED -> {
                 }
                 else -> {
-                    appLogger.e("Unknown ConnectionStatus in InternalContactSearchResultItem")
+                    appLogger.e("Unknown ConnectionStatus in InternalContactSearchResultItem $connectionState")
```

Comment:
As `connectionState` is an enum class known in runtime, I would rather add the missing `ACCEPTED` and `MISSING_LEGALHOLD_CONSENT` cases in order to be exhaustive and not have this unnecessary else.

Reply:
Good point 👍